CharlieBackend.Api/SwaggerExamples/MentorsController/GetStudentByIdResponse.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Swashbuckle.AspNetCore.Filters;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CharlieBackend.Api.SwaggerExamples.StudentsController
+{
+    class GetMentorByIdResponse : IExamplesProvider<MentorMock>
+    {
+        public MentorMock GetExamples()
+        {
+            return new MentorMock
+            {
+                First_name = "Brad",
+                Last_name = "Pitt",
+                Email = "student@example.com"
+            };
+        }
+    }
+
+    class MentorMock
```

Comment:
Why do you need this class if GetMentorById returns MentorDto?

Reply:
changed to MentorDto